bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/views/VerrechnungsDisplay.java

```diff
@@ -773,67 +777,18 @@ private boolean accept(ch.elexis.core.model.ICodeElement codeElement) {
 		@Override
 		public void dropped(List<Object> list, DropTargetEvent e) {
 			if (actEncounter != null && accept(list)) {
+				articleProcessor = new ArticleProcessor(actEncounter);
```

Comment:
Es werden nun _gedroppten_ Objekte, nicht nur Artikel, an den `ArticleProcessor` übergeben. Evtl. sollte der dann auch in `BillingProcessor` umbenannt werden.